doc/en/user/source/community/geofence/configuration.rst

```diff
@@ -0,0 +1,43 @@
+.. _geofence_configuration:
+
+GeoFence Admin GUI
+==================
+
+The GeoFence Admin Page is a component of the GeoServer web interface. You can access it from the GeoServer web interface by clicking the :guilabel:`GeoFence` link, found on the left side of the screen after logging in.
+
+.. figure:: images/configuration.png
+   :align: center
+
+General Settings
+----------------
+Configure the following settings here:
+
+- Geoserver instance name: the name under which this geoserver is known by the geofence server. This useful for when you use an external geofence server with multiple geoserver server.
+
+- GeoServer services URL: this is how geoserver knows how to connect to the external geofence server. When using an internal geofence geoserver, this is not configurable.
+
+Options
+-------
+
+Configure the following settings here:
+
+- Allow remote and inline layers in SLD
+
+- Authenticated users can write
+
+- Use GeoServer roles to get authorizations
+
+- Comma delimited list of mutually exclusive roles for authorization
+
```

Comment:
These four settings (especially the last two) probably need some more explanation - perhaps an outreference? Not a blocker for this.

Reply:
the problem is that I am not so sure myself. Perhaps someone who has better understanding of these settings can add text or clarify here? (@mbarto ?)